api/meter.go

```diff
@@ -228,6 +258,38 @@ func (r *relayMeter) AppRelays(app string, from, to time.Time) (AppRelaysRespons
 	return resp, nil
 }
 
+func (r *relayMeter) AppLatency(app string) (AppLatencyResponse, error) {
+	r.Logger.WithFields(logger.Fields{"app": app}).Info("apiserver: Received AppLatency request")
+
+	resp := AppLatencyResponse{
```

Comment:
Perhaps more readable to simply return this value instead of declaring a new variable that is used only once.

Reply:
Updated